### PR TITLE
Create single video page

### LINKED ISSUE #7: Create Single Video Page
This issue focuses on the development of a single video page.

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
created and styled a single video page.
Screenshot (126)

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Small PR looks good. No changes required

**Review by sadathshariff — APPROVED** [review]
Everything looks fine, good to go

File: src/Pages/VideoPage/VideoPage.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    
+}
+
+.txt-gray-color{
+    color: #808080;
```
Comment (aishwarya-bm) [inline src/Pages/VideoPage/VideoPage.css]: Use color variable

**Review by aishwarya-bm — APPROVED** [review]
Looks clean and readable, nice usage of utils.

File: src/Pages/VideoPage/VideoPage.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply src/Pages/VideoPage/VideoPage.css]: made the necessary changes.

**Pull request merged**